technology-considerations/internet_explorer.md

```diff
@@ -0,0 +1,40 @@
+# Developing and Testing for Internet Explorer
+
+## Problem
+
+A key deliverable for the TDR project is to support users of Internet Explorer.
```

Comment:
It's definitely worth mentioning that we know we'll have to solve Internet Explorer-specific problems (e.g. the workaround for uploading folders), so IE investigation is even more important than on a typical website where you're mostly just worrying about the CSS working cross-browser.

Reply:
Added additional comment